src/styles/StyledScrollContainer.tsx

```diff
@@ -0,0 +1,19 @@
+import Box from "@mui/material/Box";
+import { styled } from "@mui/system";
+
+export const StyledScrollContainer = styled(Box)(({ theme: { palette } }) => ({
+  overflowY: "scroll",
+  overflowX: "hidden",
+  paddingRight: "0px",
+  "&::-webkit-scrollbar": {
```

Comment:
@ostojics this is fine for right now. At some point we should also add [`scrollbar-width` and `scrollbar-color`](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Scrollbars) for Firefox support